proto-lens/src/Data/ProtoLens/TextFormat.hs

```diff
@@ -317,8 +317,8 @@ makeScalarValue BoolField (Parser.IntValue x)
 makeScalarValue DoubleField (Parser.DoubleValue x) = Right x
 makeScalarValue FloatField (Parser.DoubleValue x) = Right (realToFrac x)
 makeScalarValue BoolField (Parser.EnumValue x)
-    | x == "true" = Right True
-    | x == "false" = Right False
+    | x `elem` ["true", "True", "t"] = Right True
```

Comment:
The "1" and "0" aren't worth the test?

Reply:
"1" and "0" are handled differently; looks like they're parsed to `Int` first (see the `Parser.IntValue` match a few lines up).